src/librustc_typeck/check/wfcheck.rs

```diff
@@ -879,6 +879,11 @@ fn receiver_is_valid<'fcx, 'tcx>(
     // The first type is `receiver_ty`, which we know its not equal to `self_ty`; skip it.
     autoderef.next();
 
+    let receiver_trait_def_id = fcx.tcx.require_lang_item(
+        lang_items::ReceiverTraitLangItem,
+        None,
```

Comment:
Not `Some(span)` here?

Reply:
Nope, the span this would point to would be nonsense (it'd point to some random function that took `self: ...` as an argument)

Reply:
I see. Can you add your explanation as a comment?